src/App.js

```diff
@@ -491,9 +491,12 @@ class App extends React.Component<Props, State> {
   };
 
   onCloseOnboarding = () => {
+    const params = this.getCurrentParams();
+
     saveState({ onboardingCompleted: 'true' });
     this.setState({ isOnboardingVisible: false });
-    if (this.mainView) this.mainView.focusSearchToolbar();
+
+    this.props.routerHistory.push('search', params);
```

Comment:
How does this look in the mobile view? Will we have the full screen search popup immediately?

Reply:
🙈 You're right! Always forgetting about mobile. Will fix it.